docs/platform/howto/billing-assign-projects.rst

```diff
@@ -0,0 +1,24 @@
+Assign projects to billing groups
+=================================
+
+When you create a new project, you can assign it to a billing group, which lets you :doc:`use the same payment card and consolidate costs</docs/platform/concepts/billing-groups>`. 
+
+You can also assign the project to another billing group later in the `Aiven Console <https://console.aiven.io/>`_. 
+
+.. note::
+
+    Projects can only be assigned to one billing group. If a project is already assigned to one billing group and you assign it to another, it will be removed from the first group.
+
+To assign projects to a billing group:
+
+#. In the project, click **Billing**.
```

Comment:
How about replacing "click" with "select"? (this comment applies to the whole PR).